pkg/collector/corechecks/snmp/internal/test/test_profiles/validation_error.yaml

```diff
@@ -1,7 +1,5 @@
 # Profile for F5 BIG-IP devices
 #
-extends:
-  - does_not_exist.yaml
 
```

Comment:
note: this change is needed to avoid failing at invalid extend, but with "invalid regex" instead (see below in this file)